infra-lifecycle-wg/README.md

```diff
@@ -0,0 +1,20 @@
+# Infrastructure Lifecycle WG
+
+The [charter](./charter/charter.md) describes the goals and activites of this group.
+
+To contribute please join our Slack channel and/or biweekly meetings.
+
+#wg-infrastructure-lifecycle Slack channel: <https://cloud-native.slack.com/archives/C06USDTN683>. Get an invite at <https://slack.cncf.io/>
+
+## Chairs
+
+- Bruno Schaatsbergen (@bschaatsbergen)
+- Dylan Page (@genpage)
```

Comment:
@GenPage has agreed to withdraw his chair nomination in favour of becoming TAG TL and liaison to the WG